service-front/module/Application/view/application/authenticated/about-you/index.twig

```diff
@@ -75,15 +76,17 @@
     maxlength: '2',
     min: '0',
     max: '12',
-    pattern: '[0-9]*'
+    pattern: '[0-9]*',
+    inputmode: 'numeric'
  }) ? ''}}
 
 {{ yearOfBirth.setAttributes({
     id: 'dob-date-year',
     type: 'tel',
```

Comment:
Should this type be "tel"? I assume "tel" is for telephone numbers, but this is a year field.

Reply:
I realise you didn't add this, just wondering why it's here.

Reply:
All these fields have now been updated to Text